src/warnings.ts

```diff
@@ -1,32 +1,17 @@
-/*
- * Copyright 2019 Google LLC
- * All rights reserved.
+/**
```

Comment:
we've been using `//` comments for licenses, but if it's a hassle to make this change not a blocker for me.

Reply:
So... I first removed all the licenses using some Perl magic, then used https://github.com/google/addlicense to add them. Turns out it uses `/** */` style licenses, I guess it should be fine.